<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-8PUIHPV2" ignored="0" name="Insurence QA" tests="8" failures="2" timestamp="11 Dec 2019 07:37:43 GMT" time="77.407" errors="0">
  <testcase name="a" time="0.008" classname="com.Base.BaseTest"/>
  <testcase name="b" time="0.002" classname="com.Base.BaseTest">
    <skipped/>
  </testcase> <!-- b -->
  <testcase name="b" time="0.002" classname="com.Base.BaseTest">
    <skipped/>
  </testcase> <!-- b -->
  <testcase name="b" time="0.002" classname="com.Base.BaseTest">
    <failure type="java.lang.ArithmeticException">
      <![CDATA[java.lang.ArithmeticException
at com.Base.BaseTest.b(BaseTest.java:263)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- b -->
  <testcase name="c" time="0.002" classname="com.Base.BaseTest">
    <skipped/>
  </testcase> <!-- c -->
  <testcase name="c" time="0.002" classname="com.Base.BaseTest">
    <skipped/>
  </testcase> <!-- c -->
  <testcase name="c" time="0.001" classname="com.Base.BaseTest">
    <failure type="java.lang.ArithmeticException">
      <![CDATA[java.lang.ArithmeticException
at com.Base.BaseTest.c(BaseTest.java:268)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- c -->
  <testcase name="d" time="0.002" classname="com.Base.BaseTest"/>
</testsuite> <!-- Insurence QA -->
